app/src/main/java/com/sun/ise/data/EventDataSource.kt

```diff
@@ -6,5 +6,7 @@ import retrofit2.Call
 interface EventDataSource {
     interface Remote {
         fun getAllEvents(accessToken: String): Call<EventResult>
```

Comment:
abstract k nên phụ thuộc vào interface em ạ..
a Đã giải thích cho Cảnh, e có thể trao đổi thêm
```suggestion
        fun getAllEvents(): Call<EventResult>
```